app/src/main/java/org/sopt/korailtalk/core/designsystem/component/bottomsheet/KorailTalkBasicBottomSheet.kt

```diff
@@ -0,0 +1,106 @@
+package org.sopt.korailtalk.core.designsystem.component.bottomsheet
+
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.ColumnScope
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.WindowInsets
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.ime
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.Add
+import androidx.compose.material3.Button
+import androidx.compose.material3.Card
+import androidx.compose.material3.CardDefaults
+import androidx.compose.material3.ExperimentalMaterial3Api
+import androidx.compose.material3.ExtendedFloatingActionButton
+import androidx.compose.material3.Icon
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.ModalBottomSheet
+import androidx.compose.material3.Scaffold
+import androidx.compose.material3.Surface
+import androidx.compose.material3.Text
+import androidx.compose.material3.rememberModalBottomSheetState
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.rememberCoroutineScope
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import kotlinx.coroutines.launch
+
+/**
+ * 공통 Modal BottomSheet 컴포넌트
+ *
+ * 예제코드(사용법):
+ * ```
+ * var showSheet by remember { mutableStateOf(false) }
+ *
+ * CommonBottomSheet(
+ *     isVisible = showSheet,
+ *     onDismiss = { showSheet = false },
+ *     title = "제목"
+ * ) {
+ *     Text("내용")
+ *     Button(onClick = { }) {
+ *         Text("버튼")
+ *     }
+ * }
+ * ```
+ *
+ * @param isVisible BottomSheet 표시 여부
+ * @param onDismiss 닫기 콜백 함수
+ * @param title 제목 (선택사항)
+ * @param content BottomSheet 내부 컨텐츠
+ */
+
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+fun KorailTalkBasicBottomSheet(
+    isVisible: Boolean,
+    onDismiss: () -> Unit,
+    title: String? = null,
+    content: @Composable ColumnScope.() -> Unit
```

Comment:
유연하게 가져가려면
```suggestion
    content: @Composable () -> Unit
```
로 쓰는 게 조금 더 나을 것 같습니다